are_we/are_we.ino

```diff
@@ -0,0 +1,108 @@
+#include "FastLED.h"
+#include "CmdMessenger.h"
```

Comment:
We should think of using a more streamlined (probably custom) protocol but this is fine for now.

Reply:
I looked into passing defined structs which would be the best way to do it if we run into a serial port bottleneck, but as far as I can tell this basically implements that, just maybe not as efficient because it needs to be use-case agnostic. 

Again, fast-enough speed and ease of code writing.

Reply:
My understanding is that this uses a pure serial connection (uart?). This is much, much, slower than an equivalent SPI/I2C connection between the Pi and the arduino. As long as we do things in a mostly protocol agnostic way this should be easy to change. 

However looking again at the exact use case we may not have to worry that much about speed.

Reply:
Very true, SPI would let us get some very high throughput speeds. 

I think the bottleneck is the actual LED strip maybe? I am not sure, at 100Hz and above it starts to flicker even if no calls to a lower brightness are made. Could also be a FastLED issue but I am not sure.

Reply:
Ok we should open an issue to figure out what is causing the bottleneck in the speed. My guess is its something to do with the way FastLED does its update calls.

Reply:
Opened